docs/circuits.ipynb

```diff
@@ -742,6 +729,9 @@
    },
    "outputs": [],
    "source": [
+    "print('Original qft_3 CircuitOperation')\n",
+    "print(qft_3)\n",
+    "# No longer in a CircuitOperation.\n",
```

Comment:
Is this describing the circuits above or below the comment?

Reply:
Below. Added more context comments.